index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "extends": [
+      "tslint:latest",
+      "tslint-config-prettier"
+  ],
+  "rules": {
+      "object-literal-sort-keys": false,
```

Comment:
Only other inclusion I would suggest is: `"no-submodule-imports": false`.

Additionally I typically put a link to the rule details i.e.
```
        // https://palantir.github.io/tslint/rules/no-submodule-imports/
        "no-submodule-imports": false,
        // https://palantir.github.io/tslint/rules/object-literal-sort-keys/
        "object-literal-sort-keys": false,
        // https://palantir.github.io/tslint/rules/ordered-imports/
        "ordered-imports": false,
        // https://palantir.github.io/tslint/rules/interface-name/
        "interface-name": [
            true,
            "never-prefix"
        ],
        // https://palantir.github.io/tslint/rules/no-implicit-dependencies/
        "no-implicit-dependencies": [
            true,
            "dev"
        ]
```

Reply:
I should preface that with an example - `lodash/fp`.

I know you can add a whitelist as well but that would be too difficult to manage. I actually can't think of any others off the top of my head so not a strong case for the inclusion here.

Reply:
I’m not opposed to that. I’ve been following no submodule imports in Node but on the browser that’s putting a lot of faith in tree shaking for e.g. lodash.

Reply:
Yeah turning that off makes sense. I agree with the rule in general but too many edge cases.